src/message/MentionedUserNotSubscribed.js

```diff
@@ -67,7 +67,12 @@ class MentionedUserNotSubscribed extends PureComponent<Props> {
     const { user, onDismiss } = this.props;
     onDismiss(user);
   };
-
+  componentDidMount() {
+    LayoutAnimation.configureNext({ ...LayoutAnimation.Presets.easeInEaseOut, duration: 100 });
```

Comment:
I like that these slide in and out now! I see something a bit odd, though—it looks like the warnings are sliding *between* the compose box's background and the compose box's inputs, in the z-direction. Can we fix that? 🙂

I think it might look better if it slid in and out from under the compose box entirely, i.e., not above its background.

Here's a video with the animation slowed down so it's easier to see:

https://user-images.githubusercontent.com/22248748/111373291-1912fa80-8672-11eb-9889-4830b70265c5.mp4

Reply:
Done :) Now the Webview doesn't jump up and down and the warning overlay in the z-direction.